cloudinit/ssh_util.py

```diff
@@ -267,6 +269,21 @@ def extract_authorized_keys(username, sshd_cfg_file=DEF_SSHD_CFG):
                         "config from %r, using 'AuthorizedKeysFile' file "
                         "%r instead", DEF_SSHD_CFG, auth_key_fns[0])
 
+        # check if one of the keys is the user's one
```

Comment:
I think we can dedent this block one more level. I don't think anything here needs to be under the `with` block.